forage-android/src/main/java/com/joinforage/forage/android/ecom/ui/ForagePINEditText.kt

```diff
@@ -67,16 +67,16 @@ class ForagePINEditText @JvmOverloads constructor(
 
                     // at this point in time, we do not know the environment and
                     // we are operating and thus do not know whether to add
-                    // BTVaultWrapper, VGSVaultWrapper, or ForageVaultWrapper to the UI.
+                    // BTVaultWrapper or ForageVaultWrapper to the UI.
                     // But that's OK. We can hedge and instantiate all of them.
                     // Then, within setForageConfig, once we know the environment
                     // and are thus able to initial LaunchDarkly and find out
-                    // whether to use BT or VGS. So, below we are hedging.
+                    // whether to use BT or Forage. So, below we are hedging.
                     btVaultWrapper = BTVaultWrapper(context, attrs, defStyleAttr)
-                    vgsVaultWrapper = VGSVaultWrapper(context, attrs, defStyleAttr)
+                    forageVaultWrapper = ForageVaultWrapper(context, attrs, defStyleAttr)
                     // ensure all wrappers init with the
                     // same typeface (or the attributes)
-                    btVaultWrapper.typeface = vgsVaultWrapper.typeface
+                    forageVaultWrapper.typeface = btVaultWrapper.typeface
```

Comment:
```suggestion
                    btVaultWrapper.typeface = forageVaultWrapper.typeface
```

should it be this instead